.github/workflows/chart-lint-publish.yml

```
@@ -0,0 +1,62 @@
+        required: false
+        default: '""'
+        type: string
+      CHART_PUBLISH:
+        description: 'Chart publishing to gh-pages branch'
+        required: false
+        default: 'NO'
+        type: string
+        options:
+          - YES
+          - NO
+      INCLUDE_ALL_CHARTS:
```

actionlint (1.7.12)

[error] "options" can not be set to "include_all_charts" input because its input type is not "choice"

(events)